components/DocsHelp.tsx

```diff
@@ -250,7 +255,7 @@ export function DocsHelp() {
               target='_blank'
               rel='noreferrer'
               className='px-[16px] py-[8px] cursor-pointer border-solid border-[#aaaaaa] border rounded-md hover:bg-gray-200 dark:hover:bg-gray-600'
-              href='https://github.com/orgs/json-schema-org/projects/16'
+              href={`https://github.com/json-schema-org/website/blob/main/pages${markdownFile ? router.asPath : `/${path}.page.tsx`}.md`}
```

Comment:
I am not sure this will work for cases where we don't inform markdownFile. As a reminder, if we instance DocsHelp without informing markdownFile, it should build the link with the tsx page but the link generated in this case is wrong.

For example:
http://localhost:3000/overview/sponsors

I am creating the instance of DocsHelp with this code:
````
<DocsHelp />
````

Is generating this wrong link:
https://github.com/json-schema-org/website/blob/main/pages/%2Foverview%2Fsponsors.page.tsx.md

But it should be:
https://github.com/json-schema-org/website/blob/main/pages/overview/sponsors/index.page.tsx

Reply:
i am working on it .

Reply:
[Screencast from 2024-04-09 15-22-19.webm](https://github.com/json-schema-org/website/assets/119167601/36a224aa-fcd6-4103-99ba-acaf611a61d2)